src/libs/actions/BankAccounts.ts

```diff
@@ -55,6 +55,13 @@ function openPersonalBankAccountSetupView(exitReportID: string) {
     });
 }
 
+/**
+ * Whether after adding a bank account we should continue with the KYC flow
+ */
```

Comment:
NAB: Stores a route that is navigated to after successfully adding a bank account